src/ConfigAggregator.php

```diff
@@ -13,16 +13,11 @@
 use Laminas\Stdlib\ArrayUtils\MergeRemoveKey;
 use Laminas\Stdlib\ArrayUtils\MergeReplaceKeyInterface;
 
+use Webimpress\SafeWriter\FileWriter;
```

Comment:
minor: we should keep different types of uses together, so the empty line should be below not above this import.

Reply:
Done